rvic/core/share.py

```diff
@@ -91,68 +92,72 @@ def __init__(self,
 
         self.atts = OrderedDict()
 
-        if title:
+        if title is not None:
```

Comment:
`is not None` is more specific. This allows title to be `0`, or `False, or `''` while continuing on...